app/controllers/funding_controller.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class FundingController < ApplicationController
+  helper_method :trainee_summary_academic_years, :payment_schedule_academic_years
+
+  def show; end
+
+private
+
+  def trainee_summary_academic_years
```

Comment:
This and the method below feels like they should probably go into a view_object since they're presentational concerns eg https://github.com/DFE-Digital/register-trainee-teachers/blob/main/app/view_objects/academic_year_filter_options.rb